en/user/errors_during_fetching.md

```diff
@@ -35,6 +35,7 @@ Repeat until you have something ok.
 Then you can submit a pull request to
 [<https://github.com/fivefilters/ftr-site-config>](https://github.com/fivefilters/ftr-site-config)
 which is the global repo for siteconfig files.
+If you can't wait for update from fivefilters - you can place your config file into directory `wallabag/vendor/j0k3r/graby-site-config`.
```

Comment:
I think it's better to put `vendor/j0k3r/graby-site-config` because not everyone is using `wallabag` as the default folder